pkg/ui/src/views/cluster/containers/nodesOverview/index.tsx

```diff
@@ -153,6 +154,7 @@ class NodeList extends React.Component<LiveNodeListProps> {
         return 0;
       },
       className: "column--border-right",
+      width: "20%",
     },
     {
       key: "nodesCount",
```

Comment:
Last item on Anne's list was adjustment of this column's naming. I think that's the line below this.

Reply:
You're right. Renamed column name.